src/components/UpcomingEvents/style.module.css

```diff
@@ -0,0 +1,124 @@
+.event_page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.events {
+    margin: 0 auto;
+    font-family: Arial, sans-serif;
+    text-align: left;
+  }
+  
+  .event_card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 20px;
```

Comment:
I would avoid using margin like this to position and space out the event cards, try to use gap with flexbox